contracts/TradetrustERC721.sol

```diff
@@ -32,6 +49,12 @@ contract TradeTrustERC721 is ERC721MintableFull, IERC721Receiver {
     emit TokenBurnt(_tokenId);
   }
 
+  function sendToNewTitleEscrow(address beneficiary, address holder, uint256 _tokenId) public onlyMinter {
+    require(ownerOf(_tokenId) == address(this), "Cannot send token: Token not owned by token registry");
```

Comment:
Wondering what happens if you don't do this check. Seemed like it will revert anyway. 

![image](https://user-images.githubusercontent.com/7406870/113534051-246a9d80-9602-11eb-9246-66330f304ee1.png)

Is the purpose of this assertion to:
1. throw
2. throw a different error message than `"ERC721: transfer of token that is not own"`

Reply:
ya i agree that the extra layer of require may not be necessary since inside _transferfrom already checks it. but because the rest of the functions in TTERC721 also has this. maybe because that error message was used with the intention to directly point it to TTERC721 if it occurs? im not very sure of this part. i removed it anyway, let me know what you think and i'll change accordingly.